ops/pebble.py

```diff
@@ -3146,26 +3193,28 @@ def notify(
         Returns:
             The notice's ID.
         """
-        body: Dict[str, Any] = {
-            'action': 'add',
-            'type': type.value,
-            'key': key,
-        }
-        if data is not None:
-            body['data'] = data
-        if repeat_after is not None:
-            body['repeat-after'] = _format_timeout(repeat_after.total_seconds())
-        resp = self._request('POST', '/v1/notices', body=body)
-        return resp['result']['id']
+        with tracer.start_as_current_span('pebble notify'):
+            body: Dict[str, Any] = {
+                'action': 'add',
+                'type': type.value,
+                'key': key,
+            }
+            if data is not None:
+                body['data'] = data
+            if repeat_after is not None:
+                body['repeat-after'] = _format_timeout(repeat_after.total_seconds())
+            resp = self._request('POST', '/v1/notices', body=body)
+            return resp['result']['id']
 
     def get_notice(self, id: str) -> Notice:
         """Get details about a single notice by ID.
 
         Raises:
             APIError: if a notice with the given ID is not found (``code`` 404)
         """
-        resp = self._request('GET', f'/v1/notices/{id}')
-        return Notice.from_dict(resp['result'])
+        with tracer.start_as_current_span('pebble get_notice'):
```

Comment:
Elsewhere we're saying which service, path, etc is being worked with, so it'd seem consistent to have the notice ID in the attributes here and the filters in `get_notices` below.

Reply:
Adding now.
Original reasoning was to omit id because it felt random, that is ids cannot be correlated across traces / units / apps / models.
But now I figure it's better to log more than less.